src/control/lib/support/log.go

```diff
@@ -435,8 +438,28 @@ func getSysNameFromQuery(configPath string, log logging.Logger) ([]string, error
 	return hostNames, nil
 }
 
+func rsyncAlternateCopy(log logging.Logger, opts ...CollectLogsParams) error {
```

Comment:
From the function logic, it seems like we always expect exactly one `CollectLogsParams`.

```suggestion
func rsyncAlternateCopy(log logging.Logger, opts CollectLogsParams) error {
```

Reply:
Yes, I found this puzzling and repeated throughout the file(s).
Maybe @ravalsam can chime in here on what he was thinking?